Cargo.toml

```diff
@@ -100,26 +90,33 @@ typenum = "1"
 url = "2"
 vbs = "0.1"
 vec1 = "1.12"
+tokio = { version = "1", default-features = false, features = [
+	"rt-multi-thread",
+	"macros",
+	"parking_lot",
+	"sync",
+] }
+tracing-subscriber = "0.3"
 
 # Dependencies enabled by feature "file-system-data-source".
 atomic_store = { git = "https://github.com/EspressoSystems/atomicstore.git", tag = "0.1.4", optional = true }
 
 # Dependencies enabled by feature "sql-data-source".
 include_dir = { version = "0.7", optional = true }
-log = { version = "0.4", optional = true }
+log = { version = "0.4" }
```

Comment:
If this is required now can you move it up (out of the section labeled "Dependencies enabled by feature "sql-data-source".")

Reply:
Sure